jenkins-plugin/src/main/resources/sql/postgresql/02_migration.sql

```diff
@@ -0,0 +1,15 @@
+DROP INDEX IDX_MAVEN_ARTIFACT;
```

Comment:
because of the fix in file `01_migration.sql`

RIght now, the plugin believe your database is already in version 2, thus will not play `02_migration.sql`